assets/collage.css

```diff
@@ -200,11 +200,6 @@
     padding: 2rem;
   }
 
```

Comment:
Agreed, this is a readability issue. Though, `break-word` also presents its own set of issues when a word is broken unexpectedly. This could also cause confusion for some users.

Is there an opportunity to stack the containers instead?

Reply:
Fixed with the change in font size.